add-on/background.js

```diff
@@ -113,11 +114,34 @@ function youTubeMusicScriptThatClicksOn (actionName) {
   return '(' + script.toString().replace('kitty', actionName) + ')()'
 }
 
-async function executeGoogleMusicCommand (command) {
+function podcastsScriptThatClicksOn(actionName) {
+  const script = function (actionName) {
+
+    const audioNode = document.querySelector("audio");
```

Comment:
This most likely won't work – the UI won't be reflecting changes you applied via low level modification of audio node.
I fixed it on youtube music by actually clicking the buttons, like user would – this is the onl yway to ensure consistent user experience.

Reply:
Im not sure if you can really make it any other way, since the whole gpodcasts page is minified, and maybe obfuscated, judging from some functions i've seen. 

I've tried it in dev mode, and although it does indeed not reflect the changes in the UI, the media controls from windows (MediaSession) still reflect the changes, and the UI itself will respond correctly to the changes after any interaction (Say you play from audio node and then press the play button it will pause it and update the timeline, and so do the previous and next buttons).

I know its not perfect, but given that the extension is meant to be used while browsing other stuff, in my opinion the problem becomes minimal. 

My main approach to the problem was trying to interact with the MediaSession itself, which should handle all the calls to the main UI, but after quite some googling, i found it to be a dead end.

Reply:
I guess a good option would be to try and find these buttons using the current id's and if these fail, fall back to using the audio node. If this sounds ok to you, I should be able to implement it by tomorrow :D

Reply:
> I guess a good option would be to try and find these buttons using the current id's and if these fail, fall back to using the audio node. If this sounds ok to you, I should be able to implement it by tomorrow :D

Just applied this solution, please take a look :)

Reply:
Wow, you are right, podcasts.google.com is way more hostile to user scripts than music.google.com.
Music still had some ID elements and meaningful css class names, Podcasts are just minified nonsense. 

Makes me pretty sad about the direction of the web in general :(( 

I suggested alternative in https://github.com/lidel/google-music-hotkeys/pull/22#pullrequestreview-633008755